zonemaster/zonemaster-cli
# Updates Finnish translation (CLI)

## Purpose

This PR updates the translation (PO file) for Finnish as provided by @samisalmensuo in #264.

## Context

Resolves #264.

## How to test this PR

Change to Finnish and verify that translation comes out as expected for updated msgids and msgstrs.
